pyvrp/cpp/search/SwapStar.h

```diff
@@ -97,6 +97,13 @@ class SwapStar : public LocalSearchOperator<Route>
 
     BestMove best;
 
+    // Evaluates the delta cost for route V, when inserting U after UAfter
+    // while simultaneously removing V
+    Cost evaluateRoute(Route::Node *V,
+                       Route::Node *U,
+                       Route::Node *UAfter,
+                       CostEvaluator const &costEvaluator);
```

Comment:
It's a bit confusing that we're using U, V for both the route and the nodes here. I don't have a good solution. Maybe the rename and interpretation I suggested above helps a bit.